News/Services/NewsApiService.cs

```diff
@@ -0,0 +1,66 @@
+﻿using News.Extensions;
+using News.Helpers;
+using News.Services;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.UI.Popups;
+
+namespace News
+{
+    class NewsApiService
+    {
+
+        public async Task<List<Article>> GetNewsByTopic(string newsTopic)
+        {   
+
+            try
+            {
+                var client = new HttpClient();
```

Comment:
**Use a single instance of HttpClient for all requests (!!!)** and dispose of it in the service's destructor